src/rprocessing/applet/BuiltinApplet.java

```diff
@@ -3,6 +3,19 @@
 import org.renjin.sexp.StringVector;
 
 import processing.core.PApplet;
+import rprocessing.exception.NotFoundException;
```

Comment:
I have remove the useless import [#127(https://github.com/gaocegege/Processing.R/pull/127)

this block of import statements doesn't remove from RLangPApplet. When the RLangPApplet 	
initialize, it will be used.